ods_ci/tests/Tests/1300__model_registry/1302_model_registry_smoke.robot

```diff
@@ -18,13 +18,8 @@ Resource           ../../Resources/Common.robot
 
 
 *** Variables ***
-${MODELREGISTRY_BASE_FOLDER}=        tests/Resources/CLI/ModelRegistry
-${MODEL_REGISTRY_DB_SAMPLES}=        ${MODELREGISTRY_BASE_FOLDER}/samples/istio/mysql
-${OPERATOR_NS}                       ${OPERATOR_NAMESPACE}
-${APPLICATIONS_NS}                   ${APPLICATIONS_NAMESPACE}
-${DSC_NAME}                          default-dsc
-
-@{REDHATIO_PATH_CHECK_EXCLUSTION_LIST}    model-registry-operator-controller-manager
```

Comment:
This should be removed as MR already uses `registry.redhat.io` in RHOAI.

Reply:
reverted